docs/core/get-started.md

```diff
@@ -27,18 +27,18 @@ You can get started developing .NET Core apps by following these step-by-step tu
 * [Get Started with .NET Core and Visual Studio 2017](https://channel9.msdn.com/Blogs/dotnet/Get-Started-NET-Core-Visual-Studio-2017) - This [Channel9](https://channel9.msdn.com) video shows you how to install and use [Visual Studio 2017](https://www.visualstudio.com/), Microsoft's fully-featured IDE, to create your first cross-platform console application in .NET Core.
 * [Getting started with .NET Core using the command-line](tutorials/using-with-xplat-cli.md) - Use any code editor with the [.NET Core cross-platform command-line interface (CLI)](tools/index.md).
 
-See the [Prerequisites for Windows development](windows-prerequisites.md) topic for a list of the supported Windows versions. 
+See the [Prerequisites for Windows development](windows-prerequisites.md) topic for a list of the supported Windows versions.
 
 ## Linux
 
 Install .NET Core on your distribution/version:
 
 * [Red Hat Enterprise Linux 7 Server](https://www.microsoft.com/net/core#linuxredhat)
```

Comment:
since we cover the versions on the prereqs topics now, my suggestion would be to only list there and just include a link here.